### PR TITLE
doc: Azure doc fixes

### PR DESCRIPTION
**Pull request by umapraseeda**
Azure doc fixes

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 1]
```diff
@@ -1,86 +1,138 @@
...
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/09.png
+      :alt: 9
+
+#. Navigate to :guilabel:`All resources` in the Azure portal home page.
+
+   a. Click the function app that you created.
+   #. Select :guilabel:`Platform Settings` > :guilabel:`Authentication/Authorization` > :guilabel:`Azure Active Directory Settings`.
+
+#. Enter the B2C Application ID into the :guilabel:`Client ID` field.
+#. Enter the B2C Open ID Connect metadata URL into the :guilabel:`Issuer URI` field and click ``OK`` to finish the configuration.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
#. Enter the B2C OpenID Connect metadata URL into the :guilabel:`Issuer URI` field and click :guilabel:`OK` to finish the configuration.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 2]
```diff
@@ -1,86 +1,138 @@
...
 
-    .. image:: ./secure-azure-function-apps-with-microsoft-b2c/10.png
-       :alt: 10
+   a. In the :guilabel:`Azure AD B2C` blade, navigate to :guilabel:`User flows` under :guilabel:`Policies` and click the :guilabel:`Run user flow` button for each B2C User flow policy.
+   
+      The :guilabel:`Run user flow` blade opens for the selected user flow.
 
-#.  Validation and Testing.
-    The configuration on both the B2C application side as well as the Azure Function is now complete.
+   #. Select the entry that corresponds to your Azure function app in the :guilabel:`Application` field.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
   #. Select the entry that corresponds to your Azure Function App in the :guilabel:`Application` field.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 3]
```diff
@@ -1,86 +1,138 @@
...
-    #.  Other tests
+#. Complete the following tests:
 
-        #.  Clear browser session cookies, validate that user must authenticate when attempt to access your Azure Function.
-        #.  After successful Function App access, try opening a separate browser tab, and validate that you can automatically sign-in.
-        #.  Try to run other user flows, such as password reset or profile edit, with your Azure Function -- after the users complete these flows, the will be redirected to the Azure Function App.
+   a. Clear the browser session cookies and confirm that the user needs to authenticate during an attempt to access your Azure function.
+   #. After a successful access to the function app, open a separate browser tab and validate that you can automatically sign in.
+   #. Run other user flows, such as password reset or profile edit, with your Azure function.
+   #. After the users complete these flows, they are redirected to the Azure function app.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
   #. After the users complete these flows, they are redirected to the Azure Function App.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 4]
```diff
@@ -1,86 +1,138 @@
...
+.. _azure-adb2c:
 
-Manual Guide by `Saurabh Sharma <https://docs.microsoft.com/answers/answers/40449/view.html>`_.
+Using Azure Active Directory B2C
+################################
 
-If you want to use Azure Function Apps and need a way to secure access to your Function Apps -- that is, only allowing authenticated users, then Azure AD B2C is a great solution.
-It allows users to create their own accounts or using a social identity provider, and gain access to the your Function apps only after they have successfully authenticated.
-This document is a step-by-step guide to configuring Azure AD B2C, as the identity provider for your Azure Function Apps.
+To secure access to your Azure function apps and to allow access only to authenticated users, you can use Azure Active Directory B2C.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
To secure access to your Azure Function App and to allow access only to authenticated users, you can use Azure Active Directory B2C.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 5]
```diff
@@ -1,86 +1,138 @@
...
+It allows users to create their own accounts or sign in with credentials provided by a social identity provider.
+Users can access the function apps only after a successful authentication.
 
-#.  Create your Azure function app -- easiest way is to select **+Create a Resource** , type in "function app" to find it, select Function App, and select **Create**
+To configure Azure AD B2C as the identity provider for your function apps, complete the following steps:
 
-    .. image:: ./secure-azure-function-apps-with-microsoft-b2c/01.png
-       :alt: 1
+#. Log in to the `Azure portal <https://portal.azure.com/#home>`_.
+#. Click :guilabel:`+Create a Resource` to create your Azure function app.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
#. Click :guilabel:`+Create a Resource` to create your Azure Function App.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 6]
```diff
@@ -1,86 +1,138 @@
...
+
+   a. In the Azure Portal (open a separate browser tab or new browser session if you want the previous blade to be available), select the :guilabel:`Azure AD B2C` blade.
+   #. Select :guilabel:`Applications`, then select :guilabel:`+Add`.
+
+      The :guilabel:`New Application` blade opens.
+   #. Configure the following fields:
+	
+	  * Provide the app name.
+	  * Select ``Yes`` for the :guilabel:`Web App/ Web API` option.
+	  * In the :guilabel:`Reply URL` field, enter the unique Azure function app name that you created in step 4 and append  ``/.auth/login/aad/callback`` to it. For example, ``https://myAppSecuredByB2C.azurewebites.net/.auth/login/aad/callback``.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
	  * In the :guilabel:`Reply URL` field, enter the unique Azure Function App name that you created in step 4 and append  ``/.auth/login/aad/callback`` to it. For example, ``https://myAppSecuredByB2C.azurewebites.net/.auth/login/aad/callback``.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 7]
```diff
@@ -1,86 +1,138 @@
...
-    Complete the rest of the information and select **Create**.
+   a. Search for ``function app``.
+   #. In the ``Function App`` blade, click :guilabel:`Create`. 
 
-    .. image:: ./secure-azure-function-apps-with-microsoft-b2c/02.png
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/01.png
+      :alt: 1
+
+
+#. Enter a unique name for your function app, fill the remaining information and click :guilabel:`Create`.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
#. Enter a unique name for your Function App, fill the remaining information and click :guilabel:`Create`.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 8]
```diff
@@ -1,86 +1,138 @@
...
+
+   Note the name as you will need it in the subsequent steps.
+
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/02.png
        :alt: 2
 
-#.  Go back to the Function App blade in the Azure Portal, and select the Azure function app that you created.
-    From the Function App blade, select Platform Features and then select Authentication/Authorization.
+   Once the function app is created, click :guilabel:`Go to resource` to view the created function app.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
   Once the Function App is created, click :guilabel:`Go to resource` to view the created Function App.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 9]
```diff
@@ -1,86 +1,138 @@
...
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/03.png
+      :alt: 3
 
-#.  Move the App Service Authentication to the On Position, which then expose a number of settings available for Authentication.
-    Note that the default is "Allow Anonymous request" -- allowing any user to access your function application.
+#. In the :guilabel:`Authentication/Authorization` blade, move the sliding button under :guilabel:`App Service Authentication` to ``On``. Several authentication provider configurations are now displayed.
+    
+   .. note:: 
+	
+	  The default action when a request is not authenticated is ``Allow Anonymous request``. This allows any user to access your function app.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
	  The default action when a request is not authenticated is ``Allow Anonymous request``. This allows any user to access your Function App.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 10]
```diff
@@ -1,86 +1,138 @@
...
+
+      The :guilabel:`Run user flow` blade opens.
+   #. Copy the OpenID Metadata URL endpoint that is displayed in the top right corner.
+
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/09.png
+      :alt: 9
+
+#. Navigate to :guilabel:`All resources` in the Azure portal home page.
+
+   a. Click the function app that you created.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
   a. Click the Function App that you created.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 11]
```diff
@@ -1,86 +1,138 @@
...
+The configuration for B2C application and the Azure Function is now complete.
 
-    .. image:: ./secure-azure-function-apps-with-microsoft-b2c/09.png
-       :alt: 9
+To validate the configurations, perform the following tests:
 
-#.  Return to the Azure Function blade, and navigate to the Platform Settings -> Authentication/Authorization -> Azure Active Directory Settings -- enter the B2C Application ID into the **Client ID** property.
-    Similarly, enter in the B2C Open ID Connect metadata URL into the **Issuer URI** property.
-    Select OK to finish the configuration.
+1. Confirm that users without authentication gets prompted to complete their B2C sign-in before being allowed access to your function app:
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
1. Confirm that users without authentication gets prompted to complete their B2C sign-in before being allowed access to your Function App:
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 12]
```diff
@@ -1,86 +1,138 @@
...
+   #. Click the :guilabel:`Run user flow` button.
 
-    #.  Other tests
+#. Complete the following tests:
 
-        #.  Clear browser session cookies, validate that user must authenticate when attempt to access your Azure Function.
-        #.  After successful Function App access, try opening a separate browser tab, and validate that you can automatically sign-in.
-        #.  Try to run other user flows, such as password reset or profile edit, with your Azure Function -- after the users complete these flows, the will be redirected to the Azure Function App.
+   a. Clear the browser session cookies and confirm that the user needs to authenticate during an attempt to access your Azure function.
+   #. After a successful access to the function app, open a separate browser tab and validate that you can automatically sign in.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
   #. After a successful access to the Function App, open a separate browser tab and validate that you can automatically sign in.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 13]
```diff
@@ -1,86 +1,138 @@
...
+	
+	  The default action when a request is not authenticated is ``Allow Anonymous request``. This allows any user to access your function app.
 
-    .. image:: ./secure-azure-function-apps-with-microsoft-b2c/04.png
-       :alt: 4
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/04.png
+      :alt: 4
 
-#.  We will now select "Log in with Azure Active Directory", which is one of the available options in the drop down box labeled "Action to take when request is not authenticated"
+#. Select ``Log in with Azure Active Directory`` in the :guilabel:`Action to take when request is not authenticated` field.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
#. Select :guilabel:`Log in with Azure Active Directory` in the :guilabel:`Action to take when request is not authenticated` field.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 14]
```diff
@@ -1,86 +1,138 @@
...
 
-    .. image:: ./secure-azure-function-apps-with-microsoft-b2c/01.png
-       :alt: 1
+#. Log in to the `Azure portal <https://portal.azure.com/#home>`_.
+#. Click :guilabel:`+Create a Resource` to create your Azure function app.
 
-#.  Enter your unique Azure Function App name -- copy the name to a text editor -we will need it in subsequent steps.
-    Complete the rest of the information and select **Create**.
+   a. Search for ``function app``.
+   #. In the ``Function App`` blade, click :guilabel:`Create`.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
   #. In the :guilabel:`Function App` blade, click :guilabel:`Create`. 
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 15]
```diff
@@ -1,86 +1,138 @@
...
+#. Select :guilabel:`Authentication/Authorization`.
 
-    .. image:: ./secure-azure-function-apps-with-microsoft-b2c/03.png
-       :alt: 3
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/03.png
+      :alt: 3
 
-#.  Move the App Service Authentication to the On Position, which then expose a number of settings available for Authentication.
-    Note that the default is "Allow Anonymous request" -- allowing any user to access your function application.
+#. In the :guilabel:`Authentication/Authorization` blade, move the sliding button under :guilabel:`App Service Authentication` to ``On``. Several authentication provider configurations are now displayed.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
#. In the :guilabel:`Authentication/Authorization` blade, move the sliding button under :guilabel:`App Service Authentication` to :guilabel:`On`. Several authentication provider configurations are now displayed.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 16]
```diff
@@ -1,86 +1,138 @@
...
+#. Select ``Log in with Azure Active Directory`` in the :guilabel:`Action to take when request is not authenticated` field.
 
-    .. image:: ./secure-azure-function-apps-with-microsoft-b2c/05.png
-       :alt: 5
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/05.png
+      :alt: 5
 
-#.  After this select "Azure Active Directory" under the list of Authentication Providers, this will open a new configuration blade, select "Advanced" and note the required fields, including Client ID and Issuer Uri -- the Client Secret and Allowed Token Audiences are option fields.
-    Go to the next steps to collect the application Client ID and Issuer URI values from B2C.
+#. Select ``Azure Active Directory`` under the :guilabel:`Authentication Providers` list.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
#. Select :guilabel:`Azure Active Directory` under the :guilabel:`Authentication Providers` list.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 17]
```diff
@@ -1,86 +1,138 @@
...
+#. Create a new B2C application:
+
+   a. In the Azure Portal (open a separate browser tab or new browser session if you want the previous blade to be available), select the :guilabel:`Azure AD B2C` blade.
+   #. Select :guilabel:`Applications`, then select :guilabel:`+Add`.
+
+      The :guilabel:`New Application` blade opens.
+   #. Configure the following fields:
+	
+	  * Provide the app name.
+	  * Select ``Yes`` for the :guilabel:`Web App/ Web API` option.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
	  * Select :guilabel:`Yes` for the :guilabel:`Web App/ Web API` option.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 18]
```diff
@@ -1,86 +1,138 @@
-Using Azure AD B2C to secure your Azure Functions Apps
-######################################################
+.. _azure-adb2c:
 
-Manual Guide by `Saurabh Sharma <https://docs.microsoft.com/answers/answers/40449/view.html>`_.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst:4]: Original author removed.

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 19]
```diff
@@ -1,86 +1,138 @@
...
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/07.png
+      :alt: 7
+
+#. Determine the B2C Application ID:
+
+   a. In the Azure portal, select the :guilabel:`Azure AD B2C` blade.
+   #. Click on :guilabel:`Applications` and then click on the entry for your newly created application (``B2CFunctionApp`` in this example) to open the application profile.
+   #. Obtain the Application ID from the Properties blade of the application.
+
+   Use this ID to complete the Azure function configuration.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
   Use this ID to complete the Azure Function configuration.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 20]
```diff
@@ -1,86 +1,138 @@
...
-           :alt: 11
+   #. Click the :guilabel:`Run user flow` button.
 
-    #.  Other tests
+#. Complete the following tests:
 
-        #.  Clear browser session cookies, validate that user must authenticate when attempt to access your Azure Function.
-        #.  After successful Function App access, try opening a separate browser tab, and validate that you can automatically sign-in.
-        #.  Try to run other user flows, such as password reset or profile edit, with your Azure Function -- after the users complete these flows, the will be redirected to the Azure Function App.
+   a. Clear the browser session cookies and confirm that the user needs to authenticate during an attempt to access your Azure function.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
   a. Clear the browser session cookies and confirm that the user needs to authenticate during an attempt to access your Azure Function.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 21]
```diff
@@ -1,86 +1,138 @@
...
+   #. After a successful access to the function app, open a separate browser tab and validate that you can automatically sign in.
+   #. Run other user flows, such as password reset or profile edit, with your Azure function.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
   #. Run other user flows, such as password reset or profile edit, with your Azure Function.
```

**Review by coderbyheart — CHANGES_REQUESTED** [review]
- `Azure Function App`/`Azure Function` is a product name, capitalize in all instances
- use guilabel for all GUI elements

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 22]
```diff
@@ -1,86 +1,138 @@
...
+#. Click :guilabel:`+Create a Resource` to create your Azure function app.
 
-#.  Enter your unique Azure Function App name -- copy the name to a text editor -we will need it in subsequent steps.
-    Complete the rest of the information and select **Create**.
+   a. Search for ``function app``.
+   #. In the ``Function App`` blade, click :guilabel:`Create`. 
 
-    .. image:: ./secure-azure-function-apps-with-microsoft-b2c/02.png
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/01.png
+      :alt: 1
```
Comment (peknis) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]: Do we want to have a caption for the image?

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 23]
```diff
@@ -1,86 +1,138 @@
...
-    From the Function App blade, select Platform Features and then select Authentication/Authorization.
+   Once the function app is created, click :guilabel:`Go to resource` to view the created function app.
+
+#. Select :guilabel:`Platform Features` tab.
+#. Select :guilabel:`Authentication/Authorization`.
 
-    .. image:: ./secure-azure-function-apps-with-microsoft-b2c/03.png
-       :alt: 3
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/03.png
+      :alt: 3
```
Comment (peknis) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]: Caption?

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 24]
```diff
@@ -1,86 +1,138 @@
...
-    .. image:: ./secure-azure-function-apps-with-microsoft-b2c/06.png
+   A new configuration blade called :guilabel:`Azure Active Directory Settings` opens.
+	
+#. Select :guilabel:`Advanced` as the :guilabel:`Management mode`.
+
+   a. Fill in :guilabel:`Client ID` and :guilabel:`Issuer Url`.
+   #. Optionally, fill in :guilabel:`Client Secret` and :guilabel:`Allowed Token Audiences`. Click :guilabel:`OK`.
+
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/06.png
        :alt: 6
```
Comment (peknis) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst:40]: Caption?

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 25]
```diff
@@ -1,86 +1,138 @@
...
+   #. Configure the following fields:
+	
+	  * Provide the app name.
+	  * Select ``Yes`` for the :guilabel:`Web App/ Web API` option.
+	  * In the :guilabel:`Reply URL` field, enter the unique Azure function app name that you created in step 4 and append  ``/.auth/login/aad/callback`` to it. For example, ``https://myAppSecuredByB2C.azurewebites.net/.auth/login/aad/callback``.
+   #. Click :guilabel:`Create`.
+
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/07.png
+      :alt: 7
+
```
Comment (peknis) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]: Caption?

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 26]
```diff
@@ -1,86 +1,138 @@
...
+#. Determine the B2C Application ID:
+
+   a. In the Azure portal, select the :guilabel:`Azure AD B2C` blade.
+   #. Click on :guilabel:`Applications` and then click on the entry for your newly created application (``B2CFunctionApp`` in this example) to open the application profile.
+   #. Obtain the Application ID from the Properties blade of the application.
```
Comment (peknis) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
   #. Obtain the Application ID from the :guilabel:`Properties` blade of the application.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 27]
```diff
@@ -1,86 +1,138 @@
...
+
+   Use this ID to complete the Azure function configuration.
+
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/08.png
+      :alt: 8
+
```
Comment (peknis) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst:64]: Caption?

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 28]
```diff
@@ -1,86 +1,138 @@
...
 
-    .. image:: ./secure-azure-function-apps-with-microsoft-b2c/07.png
-       :alt: 7
 
-#.  Collect B2C Application ID.
-    After successfully creating the application, go back to the B2C Admin Portal Application blade, find your newly created application and open the profile to collect the Application ID -- copy this B2C application ID -- this will be used to finish the Azure Function configuration.
+#. Create a new B2C application:
+
+   a. In the Azure Portal (open a separate browser tab or new browser session if you want the previous blade to be available), select the :guilabel:`Azure AD B2C` blade.
+   #. Select :guilabel:`Applications`, then select :guilabel:`+Add`.
```
Comment (peknis) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]:
```suggestion
   #. Select :guilabel:`Applications` > :guilabel:`+Add`.
```

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 29]
```diff
@@ -1,86 +1,138 @@
...
+   a. Click the function app that you created.
+   #. Select :guilabel:`Platform Settings` > :guilabel:`Authentication/Authorization` > :guilabel:`Azure Active Directory Settings`.
+
+#. Enter the B2C Application ID into the :guilabel:`Client ID` field.
+#. Enter the B2C Open ID Connect metadata URL into the :guilabel:`Issuer URI` field and click ``OK`` to finish the configuration.
 
-    .. image:: ./secure-azure-function-apps-with-microsoft-b2c/08.png
-       :alt: 8
+   .. image:: ./secure-azure-function-apps-with-microsoft-b2c/10.png
+      :alt: 10
```
Comment (peknis) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]: Caption?

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 30]
```diff
@@ -1,86 +1,138 @@
...
-    The configuration on both the B2C application side as well as the Azure Function is now complete.
+   #. Select the entry that corresponds to your Azure function app in the :guilabel:`Application` field.
+   #. Select the associated :guilabel:`Reply URL` (in some cases you might have more than one reply URL).
 
-    #.  Attempting to access your Function App will now require authentication.
-        Validate that un-authenticated users will get prompted to complete their B2C sign-in before being allowed access to your Function App.
-        This can be easily executed from the Run userflow button from each B2C User Flow Policy.
-        Make sure to select the Application representing your Azure Function App in the Application drop down picker dialog, and the associated Reply URL (in some cases you many have more than one reply URL).
+      .. image:: ./secure-azure-function-apps-with-microsoft-b2c/11.png
+         :alt: 11
```
Comment (peknis) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst:113]: Caption?

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 31]
```diff
@@ -1,86 +1,138 @@
...
 
-If you want to use Azure Function Apps and need a way to secure access to your Function Apps -- that is, only allowing authenticated users, then Azure AD B2C is a great solution.
-It allows users to create their own accounts or using a social identity provider, and gain access to the your Function apps only after they have successfully authenticated.
-This document is a step-by-step guide to configuring Azure AD B2C, as the identity provider for your Azure Function Apps.
+To secure access to your Azure function apps and to allow access only to authenticated users, you can use Azure Active Directory B2C.
+It allows users to create their own accounts or sign in with credentials provided by a social identity provider.
+Users can access the function apps only after a successful authentication.
 
-#.  Create your Azure function app -- easiest way is to select **+Create a Resource** , type in "function app" to find it, select Function App, and select **Create**
+To configure Azure AD B2C as the identity provider for your function apps, complete the following steps:
```
Comment (peknis) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]: Another instance of "function apps".

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 32]
```diff
@@ -1,86 +1,138 @@
...
-Manual Guide by `Saurabh Sharma <https://docs.microsoft.com/answers/answers/40449/view.html>`_.
+Using Azure Active Directory B2C
+################################
 
-If you want to use Azure Function Apps and need a way to secure access to your Function Apps -- that is, only allowing authenticated users, then Azure AD B2C is a great solution.
-It allows users to create their own accounts or using a social identity provider, and gain access to the your Function apps only after they have successfully authenticated.
-This document is a step-by-step guide to configuring Azure AD B2C, as the identity provider for your Azure Function Apps.
+To secure access to your Azure function apps and to allow access only to authenticated users, you can use Azure Active Directory B2C.
+It allows users to create their own accounts or sign in with credentials provided by a social identity provider.
+Users can access the function apps only after a successful authentication.
```
Comment (peknis) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst]: Do we need to capitalize also this instance of "function apps"?

**Review by peknis — CHANGES_REQUESTED** [review]

**Comment by wiba-nordic** [comment]
I took out the illustrations, because when I was following the instructions, they weren't really helping and they were out of date with the updated Azure UI.

**Comment by peknis** [comment]
The doc has now been stripped off all images. A few images might help the reader to navigate, and at least gives a bit of breath in a long procedure like this. Also, check again that all instances of Function App are capitalized.

**Comment by peknis** [comment]
> * `Azure Function App`/`Azure Function` is a product name, capitalize in all instances
> * use guilabel for all GUI elements

However, one could argue whether the Function App needs to be capitalized or not. See for example MS's own docs: https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-function-app-portal

File: docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst [hunk 33]
```diff
@@ -1,86 +1,124 @@
...
 
-    .. image:: ./secure-azure-function-apps-with-microsoft-b2c/01.png
-       :alt: 1
+1. Log in to the `Azure portal <https://portal.azure.com/#home>`_.
+#. Click :guilabel:`Create a resource` to create your Azure Function App.
 
-#.  Enter your unique Azure Function App name -- copy the name to a text editor -we will need it in subsequent steps.
-    Complete the rest of the information and select **Create**.
+   a. Search for ``function app``.
+   #. Under ``Function App``, click :guilabel:`Create`.
```
Comment (coderbyheart) [inline docs/azure/secure-azure-function-apps-with-microsoft-b2c.rst:16]:
```suggestion
   #. Under :guilabel:`Function App`, click :guilabel:`Create`.
```

**Review by coderbyheart — APPROVED** [review]

**Review by peknis — APPROVED** [review]

**Pull request merged**